assets/setup-wizard/constants.js

```diff
@@ -0,0 +1 @@
+export const HOME_PATH = 'admin.php?page=sensei-home';
```

Comment:
I was about to comment about this, given that I think a `/wp-admin/` prefixed link wouldn't work with multisite installations using subdirs. :joy: 

Good catch! :)